rust/src/importer.rs

```diff
@@ -251,6 +254,43 @@ impl RpmImporter {
             x => unreachable!("unknown commit result '{}' for path '{}'", x, path),
         }
     }
+
+    /// Translate a filepath to an equivalent tmpfiles.d line.
+    pub fn translate_to_tmpfiles_entry(
+        &mut self,
+        abs_path: &str,
+        mut file_info: Pin<&mut crate::FFIGFileInfo>,
```

Comment:
This can be just `&crate::FFIGFileInfo` now; see https://github.com/coreos/rpm-ostree/pull/3723